docarray/array/storage/base/helper.py

```diff
@@ -0,0 +1,57 @@
+from typing import Iterator, List, Tuple, Dict
+
+
+class Offset2ID:
+    def __init__(self):
+        self.offset2id = []
```

Comment:
weird name. `Offset2ID` is the name. So this should be simply named `container` or `ids`

Reply:
renamed to `ids`